lib/internal/cluster/master.js

```diff
@@ -70,6 +79,15 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
+  const hasDebugArg = process.execArgv.some((argv) => {
```

Comment:
What the hell "Enbale" stands for? :'D
I thought you've made a typo.. but no!
<img width="668" alt="2017-06-05 1 21 31" src="https://cloud.githubusercontent.com/assets/588262/26765856/64efe60a-498d-11e7-99e0-1323b1895771.png">